outermysite/polls/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+        url(r'^search/', views.search_movie, name='search_movie'),
+        url(r'^exact/', views.exact_movie, name='exact_movie'),
```

Comment:
you could make your URLs look better doing some tweaks to make them look more like REST APIs,

See an example of something similar to what you want to do - https://dev.twitter.com/rest/reference/get/users/search